CHANGELOG.md

```diff
@@ -5,6 +5,90 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.10.2
+_Aug 20, 2025_
+
+We'd like to extend a big thank you to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌎 Improve finnish (fi-FI) locale on the Data Grid
+
+Special thanks go out to the community members for their valuable contributions:
+@lauri-heinonen-2025-04, @Methuselah96, @sai6855, @wilcoschoneveld
```

Comment:
not sure why sai has 'mui' as org ... is he one of the alumni @oliviertassinari?

Reply:
We have them in https://www.notion.so/mui-org/Supported-contributors-89b15b8d25f04f68af6e9300a2712aeb?p=394bb4ca62a547d08c30a7544192dc42